src/lib/components/payload-input-with-encoding.svelte

```diff
@@ -0,0 +1,61 @@
+<script context="module" lang="ts">
+  const encoding = ['json/plain', 'json/protobuf'] as const;
+  export type PayloadInputEncoding = (typeof encoding)[number];
+  export const isPayloadInputEncodingType = (
+    x: any, // eslint-disable-line @typescript-eslint/no-explicit-any
```

Comment:
I almost _always_ go with `unknown` over `any`. It doesn't really matter in this case, but it will at least make `eslint` happier.

Reply:
I tried `unknown` first but was getting the error `Argument of type 'unknown' is not assignable to parameter of type '"json/plain" | "json/protobuf"'.`

Maybe if I just need to add an `as` (e.g. `encoding.includes(x as PayloadInputEncoding)`) 🤔